package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "biome check --apply . --diagnostic-level=warn",
     "start": "next start",
     "typecheck": "tsc --project tsconfig.json --noEmit",
-    "update-dependencies": "bun ./scripts/update-dependencies.ts"
+    "preinstall": "npm_config_yes=true npm exec --no-package-lock=true only-allow@latest bun"
```

Comment:
this will error out of someone tries to use npm or other package managers instead of bun and show this error message:
![CleanShot 2024-10-23 at 16 19 39@2x](https://github.com/user-attachments/assets/c6b95035-790a-4de2-bc40-f6cc59e5addd)

Reply:
Thanks! Looks good, and update-dependencies was not needed anymore as you have recently removed it.